src/lib.rs

```diff
@@ -133,6 +146,7 @@ pub fn external_balance(address: &[u8; 20]) -> [u8; 16] {
     ret
 }
 
+/// Returns the beneficiary address for the most recent block mined.
```

Comment:
I would rephrase these to read "the block this transaction is in (current block)"